packages/devextreme-scss/scss/widgets/fluent/chat/_index.scss

```diff
@@ -1,3 +1,38 @@
 @use "../../base/chat";
+@use "colors" as *;
+@use "sizes" as *;
+@use "../../base/chat/mixins" as *;
 
 // adduse
+
+@include chat(
+  $chat-background-color,
+  $chat-border-color,
+  $chat-border-radius
+);
+@include chat-avatar($chat-avatar-size, $chat-avatar-background-color);
+@include chat-header($chat-header-padding);
+@include chat-messagebox(
+  $chat-messagebox-padding,
+  $chat-messagebox-border-color,
+  $chat-messagebox-gap,
+  $chat-messagebox-textarea-border-radius,
+  $chat-messagebox-buton-gap
+);
+@include chat-messagebubble(
+  $chat-bubble-padding,
+  $chat-bubble-border-radius,
+  $chat-bubble-background-color-secondary,
+  $chat-bubble-background-color-primary,
+  $chat-bubble-container-gap
+);
+@include chat-messagegroup(
+  $chat-messagegroup-gap,
+  $chat-messagegroup-alignment-start-template,
+  $chat-messagegroup-alignment-start-gap,
+  $chat-messagegroup-padding,
```

Comment:
We have a task to fix the avatar position, it involves me changing display: grid to display: flex. In this case the extra indentation will disappear. It will be completed today.